testapi.pm

```diff
@@ -903,12 +903,15 @@ fetches the script through HTTP into the SUT and execs it with C<bash -xe> and d
 C<stdout> (I<not> C<stderr>!) to the serial console and returns the output I<if> the script
 exists with 0. Otherwise the test is set to failed.
 
-The default timeout for the script is 30 seconds. If you need more, pass a second parameter
+The default timeout for the script is 30 seconds. If you need more, pass a second parameter.
+By passing as third parameter "offline => 1" you workaround case when SUT don't have network -
```

Comment:
Better use `C<…>` for escaping and also update L900. Also for better english I recommend "Pass as third parameter C<offline => 1> in case the SUT does not have network. Instead of downloading the script this function will just use C<type_string> to type the passed script string."

Reply:
"C<oofline => 1>" - will not work because "=>" need to be escaped somehow , do you know how to escape here ? 
And how I can check resulted text BTW ?